### ISSUE TITLE
This command "conda install nvidia/label/cuda-xxx::cuda" install cuda 12.4 even though other version is specified.

### ISSUE THREAD

**Issue by vgthengane** [ContinuumIO/anaconda-issues#13398]
### Checklist

- [X] I added a descriptive title.
- [X] I searched through [existing issues](https://github.com/ContinuumIO/anaconda-issues/issues) and couldn't find a solution or duplicate issue.
- [X] I searched on the web (e.g. Google) and didn't find any helpful information.
- [X] I searched the [Anaconda documentation](https://docs.anaconda.com/) and didn't find any helpful information.

### Impacted product

- [ ] A conda package (built by Anaconda)
- [ ] Anaconda Distribution (formerly Anaconda Individual Edition)
- [X] Miniconda
- [ ] Anaconda.org

### What happened?

Command - `conda install nvidia/label/cuda-11.8.0::cuda`

### Un-Expected behavior

```
Channels:
 - defaults
 - nvidia/label/cuda-11.8.0
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /vol/research/Vishal_Thengane/miniconda3/envs/point-xlstm

  added / updated specs:
    - nvidia/label/cuda-11.8.0::cuda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    cuda-runtime-12.4.1        |       hb982923_0          18 KB
    cuda-toolkit-12.4.1        |       hb982923_0          19 KB
    ------------------------------------------------------------
                                           Total:          37 KB

The following NEW packages will be INSTALLED:

  _sysroot_linux-64~ pkgs/main/noarch::_sysroot_linux-64_curr_repodata_hack-3-haa98f57_10 
  binutils           pkgs/main/linux-64::binutils-2.38-h1680402_1 
  binutils_impl_lin~ pkgs/main/linux-64::binutils_impl_linux-64-2.38-h2a08ee3_1 
  binutils_linux-64  pkgs/main/linux-64::binutils_linux-64-2.38.0-hc2dff05_0 
  bzip2              pkgs/main/linux-64::bzip2-1.0.8-h5eee18b_6 
  cccl               pkgs/main/linux-64::cccl-2.3.2-h2c7f797_0 
  cuda               nvidia/label/cuda-11.8.0/linux-64::cuda-11.8.0-0 
  cuda-cccl          pkgs/main/linux-64::cuda-cccl-12.4.127-h06a4308_2 
  cuda-cccl_linux-64 pkgs/main/noarch::cuda-cccl_linux-64-12.4.127-h06a4308_2 
  cuda-command-line~ pkgs/main/linux-64::cuda-command-line-tools-12.4.1-h06a4308_1 
  cuda-compiler      pkgs/main/linux-64::cuda-compiler-12.4.1-h6a678d5_1 
  cuda-crt-dev_linu~ pkgs/main/noarch::cuda-crt-dev_linux-64-12.4.131-h06a4308_0 
  cuda-crt-tools     pkgs/main/linux-64::cuda-crt-tools-12.4.131-h06a4308_0 
  cuda-cudart        pkgs/main/linux-64::cuda-cudart-12.4.127-h99ab3db_0 
  cuda-cudart-dev    pkgs/main/linux-64::cuda-cudart-dev-12.4.127-h99ab3db_0 
  cuda-cudart-dev_l~ pkgs/main/noarch::cuda-cudart-dev_linux-64-12.4.127-hd681fbe_0 
  cuda-cudart-static pkgs/main/linux-64::cuda-cudart-static-12.4.127-h99ab3db_0 
  cuda-cudart-stati~ pkgs/main/noarch::cuda-cudart-static_linux-64-12.4.127-hd681fbe_0 
  cuda-cudart_linux~ pkgs/main/noarch::cuda-cudart_linux-64-12.4.127-hd681fbe_0 
  cuda-cuobjdump     pkgs/main/linux-64::cuda-cuobjdump-12.4.127-h6a678d5_1 
  cuda-cupti         pkgs/main/linux-64::cuda-cupti-12.4.127-h6a678d5_1 
  cuda-cupti-dev     pkgs/main/linux-64::cuda-cupti-dev-12.4.127-h6a678d5_1 
  cuda-cuxxfilt      pkgs/main/linux-64::cuda-cuxxfilt-12.4.127-h6a678d5_1 
  cuda-demo-suite    nvidia/label/cuda-11.8.0/linux-64::cuda-demo-suite-11.8.86-0 
  cuda-driver-dev    pkgs/main/linux-64::cuda-driver-dev-12.4.127-h99ab3db_0 
  cuda-driver-dev_l~ pkgs/main/noarch::cuda-driver-dev_linux-64-12.4.127-hd681fbe_0 
  cuda-gdb           pkgs/main/linux-64::cuda-gdb-12.4.127-h122497a_1 
  cuda-libraries     pkgs/main/linux-64::cuda-libraries-12.4.1-h06a4308_1 
  cuda-libraries-dev pkgs/main/linux-64::cuda-libraries-dev-12.4.1-h06a4308_1 
  cuda-nsight        pkgs/main/linux-64::cuda-nsight-12.4.127-h06a4308_1 
  cuda-nvcc          pkgs/main/linux-64::cuda-nvcc-12.4.131-h02f8991_0 
  cuda-nvcc-dev_lin~ pkgs/main/noarch::cuda-nvcc-dev_linux-64-12.4.131-h4ee8466_0 
  cuda-nvcc-impl     pkgs/main/linux-64::cuda-nvcc-impl-12.4.131-h99ab3db_0 
  cuda-nvcc-tools    pkgs/main/linux-64::cuda-nvcc-tools-12.4.131-h99ab3db_0 
  cuda-nvcc_linux-64 pkgs/main/linux-64::cuda-nvcc_linux-64-12.4.131-he92618c_0 
  cuda-nvdisasm      pkgs/main/linux-64::cuda-nvdisasm-12.4.127-h6a678d5_1 
  cuda-nvml-dev      pkgs/main/linux-64::cuda-nvml-dev-12.4.127-h6a678d5_1 
  cuda-nvprof        pkgs/main/linux-64::cuda-nvprof-12.4.127-h6a678d5_1 
  cuda-nvprune       pkgs/main/linux-64::cuda-nvprune-12.4.127-h6a678d5_1 
  cuda-nvrtc         pkgs/main/linux-64::cuda-nvrtc-12.4.127-h99ab3db_1 
  cuda-nvrtc-dev     pkgs/main/linux-64::cuda-nvrtc-dev-12.4.127-h99ab3db_1 
  cuda-nvtx          pkgs/main/linux-64::cuda-nvtx-12.4.127-h6a678d5_1 
  cuda-nvvm-dev_lin~ pkgs/main/noarch::cuda-nvvm-dev_linux-64-12.4.131-h06a4308_0 
  cuda-nvvm-impl     pkgs/main/linux-64::cuda-nvvm-impl-12.4.131-h6a678d5_0 
  cuda-nvvm-tools    pkgs/main/linux-64::cuda-nvvm-tools-12.4.131-h6a678d5_0 
  cuda-nvvp          pkgs/main/linux-64::cuda-nvvp-12.4.127-h6a678d5_1 
  cuda-opencl        pkgs/main/linux-64::cuda-opencl-12.4.127-h6a678d5_0 
  cuda-opencl-dev    pkgs/main/linux-64::cuda-opencl-dev-12.4.127-h6a678d5_0 
  cuda-profiler-api  pkgs/main/linux-64::cuda-profiler-api-12.4.127-h06a4308_1 
  cuda-runtime       pkgs/main/noarch::cuda-runtime-12.4.1-hb982923_0 
  cuda-sanitizer-api pkgs/main/linux-64::cuda-sanitizer-api-12.4.127-h99ab3db_1 
  cuda-toolkit       pkgs/main/noarch::cuda-toolkit-12.4.1-hb982923_0 
  cuda-tools         pkgs/main/linux-64::cuda-tools-12.4.1-h06a4308_1 
  cuda-version       pkgs/main/noarch::cuda-version-12.4-hbda6634_3 
  cuda-visual-tools  pkgs/main/linux-64::cuda-visual-tools-12.4.1-h06a4308_1 
  dbus               pkgs/main/linux-64::dbus-1.13.18-hb2f20db_0 
  expat              pkgs/main/linux-64::expat-2.6.2-h6a678d5_0 
  fontconfig         pkgs/main/linux-64::fontconfig-2.14.1-h4c34cd2_2 
  freetype           pkgs/main/linux-64::freetype-2.12.1-h4a9f257_0 
  gcc_impl_linux-64  pkgs/main/linux-64::gcc_impl_linux-64-11.2.0-h1234567_1 
  gcc_linux-64       pkgs/main/linux-64::gcc_linux-64-11.2.0-h5c386dc_0 
  gds-tools          pkgs/main/linux-64::gds-tools-1.9.1.3-h99ab3db_1 
  glib               pkgs/main/linux-64::glib-2.78.4-h6a678d5_0 
  glib-tools         pkgs/main/linux-64::glib-tools-2.78.4-h6a678d5_0 
  gmp                pkgs/main/linux-64::gmp-6.2.1-h295c915_3 
  gxx_impl_linux-64  pkgs/main/linux-64::gxx_impl_linux-64-11.2.0-h1234567_1 
  gxx_linux-64       pkgs/main/linux-64::gxx_linux-64-11.2.0-hc2dff05_0 
  icu                pkgs/main/linux-64::icu-73.1-h6a678d5_0 
  kernel-headers_li~ pkgs/main/noarch::kernel-headers_linux-64-3.10.0-h57e8cba_10 
  krb5               pkgs/main/linux-64::krb5-1.20.1-h143b758_1 
  libcublas          pkgs/main/linux-64::libcublas-12.4.5.8-h99ab3db_1 
  libcublas-dev      pkgs/main/linux-64::libcublas-dev-12.4.5.8-h99ab3db_1 
  libcufft           pkgs/main/linux-64::libcufft-11.2.1.3-h99ab3db_1 
  libcufft-dev       pkgs/main/linux-64::libcufft-dev-11.2.1.3-h99ab3db_1 
  libcufile          pkgs/main/linux-64::libcufile-1.9.1.3-h99ab3db_1 
  libcufile-dev      pkgs/main/linux-64::libcufile-dev-1.9.1.3-h99ab3db_1 
  libcurand          pkgs/main/linux-64::libcurand-10.3.5.147-h99ab3db_1 
  libcurand-dev      pkgs/main/linux-64::libcurand-dev-10.3.5.147-h99ab3db_1 
  libcusolver        pkgs/main/linux-64::libcusolver-11.6.1.9-h99ab3db_1 
  libcusolver-dev    pkgs/main/linux-64::libcusolver-dev-11.6.1.9-h99ab3db_1 
  libcusparse        pkgs/main/linux-64::libcusparse-12.3.1.170-h99ab3db_1 
  libcusparse-dev    pkgs/main/linux-64::libcusparse-dev-12.3.1.170-h99ab3db_1 
  libedit            pkgs/main/linux-64::libedit-3.1.20230828-h5eee18b_0 
  libgcc-devel_linu~ pkgs/main/linux-64::libgcc-devel_linux-64-11.2.0-h1234567_1 
  libglib            pkgs/main/linux-64::libglib-2.78.4-hdc74915_0 
  libiconv           pkgs/main/linux-64::libiconv-1.16-h5eee18b_3 
  libnpp             pkgs/main/linux-64::libnpp-12.2.5.30-h99ab3db_1 
  libnpp-dev         pkgs/main/linux-64::libnpp-dev-12.2.5.30-h99ab3db_1 
  libnvfatbin        pkgs/main/linux-64::libnvfatbin-12.4.127-h7934f7d_2 
  libnvfatbin-dev    pkgs/main/linux-64::libnvfatbin-dev-12.4.127-h7934f7d_2 
  libnvjitlink       pkgs/main/linux-64::libnvjitlink-12.4.127-h99ab3db_1 
  libnvjitlink-dev   pkgs/main/linux-64::libnvjitlink-dev-12.4.127-h99ab3db_1 
  libnvjpeg          pkgs/main/linux-64::libnvjpeg-12.3.1.117-h6a678d5_1 
  libnvjpeg-dev      pkgs/main/linux-64::libnvjpeg-dev-12.3.1.117-h06a4308_1 
  libpng             pkgs/main/linux-64::libpng-1.6.39-h5eee18b_0 
  libstdcxx-devel_l~ pkgs/main/linux-64::libstdcxx-devel_linux-64-11.2.0-h1234567_1 
  libuuid            pkgs/main/linux-64::libuuid-1.41.5-h5eee18b_0 
  libxcb             pkgs/main/linux-64::libxcb-1.15-h7f8727e_0 
  libxkbcommon       pkgs/main/linux-64::libxkbcommon-1.0.1-h5eee18b_1 
  libxml2            pkgs/main/linux-64::libxml2-2.10.4-hfdd30dd_2 
  nsight-compute     pkgs/main/linux-64::nsight-compute-2024.1.1.4-h968f9c8_2 
  nspr               pkgs/main/linux-64::nspr-4.35-h6a678d5_0 
  nss                pkgs/main/linux-64::nss-3.89.1-h6a678d5_0 
  ocl-icd            pkgs/main/linux-64::ocl-icd-2.3.2-h5eee18b_1 
  pcre2              pkgs/main/linux-64::pcre2-10.42-hebb0a14_1 
  sysroot_linux-64   pkgs/main/noarch::sysroot_linux-64-2.17-h57e8cba_10 


Proceed ([y]/n)? y


Downloading and Extracting Packages:
                                                                                                                                                                                                   
Preparing transaction: done                                                                                                                                                                        
Verifying transaction: done
Executing transaction: done
```

nvcc --version

```
nvcc warning : incompatible redefinition for option 'compiler-bindir', the last value of this option was used
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2024 NVIDIA Corporation
Built on Thu_Mar_28_02:18:24_PDT_2024
Cuda compilation tools, release 12.4, V12.4.131
Build cuda_12.4.r12.4/compiler.34097967_0
```

### Conda info

```shell
active environment : envname
    active env location : Userdir/miniconda3/envs/envname
            shell level : 2
       user config file : storagedir/.condarc
 populated config files : storagedir/.condarc
          conda version : 24.1.2
    conda-build version : not installed
         python version : 3.12.1.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=skylake
                          __conda=24.1.2=0
                          __cuda=12.4=0
                          __glibc=2.35=0
                          __linux=5.15.0=0
                          __unix=0=0
       base environment : Userdir/miniconda3  (writable)
      conda av data dir : Userdir/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : Userdir/miniconda3/pkgs
                          storagedir/.conda/pkgs
       envs directories : Userdir/miniconda3/envs
                          storagedir/.conda/envs
               platform : linux-64
             user-agent : conda/24.1.2 requests/2.31.0 CPython/3.12.1 Linux/5.15.0-112-generic ubuntu/22.04.4 glibc/2.35 solver/libmamba conda-libmamba-solver/23.12.0 libmambapy/1.5.3
                UID:GID : 331909:5000093
             netrc file : storagedir/.netrc
           offline mode : False
```

### Conda config

```shell
==> storagedir/.condarc <==
auto_activate_base: False
```

### Conda list

```shell
# packages in environment at Userdir/miniconda3/envs/envname:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
_openmp_mutex             5.1                       1_gnu    defaults
_sysroot_linux-64_curr_repodata_hack 3                   haa98f57_10    defaults
binutils                  2.38                 h1680402_1    defaults
binutils_impl_linux-64    2.38                 h2a08ee3_1    defaults
binutils_linux-64         2.38.0               hc2dff05_0    defaults
bzip2                     1.0.8                h5eee18b_6    defaults
ca-certificates           2024.3.11            h06a4308_0    defaults
cccl                      2.3.2                h2c7f797_0    defaults
cuda                      11.8.0                        0    nvidia/label/cuda-11.8.0
cuda-cccl                 12.4.127             h06a4308_2    defaults
cuda-cccl_linux-64        12.4.127             h06a4308_2    defaults
cuda-command-line-tools   12.4.1               h06a4308_1    defaults
cuda-compiler             12.4.1               h6a678d5_1    defaults
cuda-crt-dev_linux-64     12.4.131             h06a4308_0    defaults
cuda-crt-tools            12.4.131             h06a4308_0    defaults
cuda-cudart               12.4.127             h99ab3db_0    defaults
cuda-cudart-dev           12.4.127             h99ab3db_0    defaults
cuda-cudart-dev_linux-64  12.4.127             hd681fbe_0    defaults
cuda-cudart-static        12.4.127             h99ab3db_0    defaults
cuda-cudart-static_linux-64 12.4.127             hd681fbe_0    defaults
cuda-cudart_linux-64      12.4.127             hd681fbe_0    defaults
cuda-cuobjdump            12.4.127             h6a678d5_1    defaults
cuda-cupti                12.4.127             h6a678d5_1    defaults
cuda-cupti-dev            12.4.127             h6a678d5_1    defaults
cuda-cuxxfilt             12.4.127             h6a678d5_1    defaults
cuda-demo-suite           11.8.86                       0    nvidia/label/cuda-11.8.0
cuda-driver-dev           12.4.127             h99ab3db_0    defaults
cuda-driver-dev_linux-64  12.4.127             hd681fbe_0    defaults
cuda-gdb                  12.4.127             h122497a_1    defaults
cuda-libraries            12.4.1               h06a4308_1    defaults
cuda-libraries-dev        12.4.1               h06a4308_1    defaults
cuda-nsight               12.4.127             h06a4308_1    defaults
cuda-nvcc                 12.4.131             h02f8991_0    defaults
cuda-nvcc-dev_linux-64    12.4.131             h4ee8466_0    defaults
cuda-nvcc-impl            12.4.131             h99ab3db_0    defaults
cuda-nvcc-tools           12.4.131             h99ab3db_0    defaults
cuda-nvcc_linux-64        12.4.131             he92618c_0    defaults
cuda-nvdisasm             12.4.127             h6a678d5_1    defaults
cuda-nvml-dev             12.4.127             h6a678d5_1    defaults
cuda-nvprof               12.4.127             h6a678d5_1    defaults
cuda-nvprune              12.4.127             h6a678d5_1    defaults
cuda-nvrtc                12.4.127             h99ab3db_1    defaults
cuda-nvrtc-dev            12.4.127             h99ab3db_1    defaults
cuda-nvtx                 12.4.127             h6a678d5_1    defaults
cuda-nvvm-dev_linux-64    12.4.131             h06a4308_0    defaults
cuda-nvvm-impl            12.4.131             h6a678d5_0    defaults
cuda-nvvm-tools           12.4.131             h6a678d5_0    defaults
cuda-nvvp                 12.4.127             h6a678d5_1    defaults
cuda-opencl               12.4.127             h6a678d5_0    defaults
cuda-opencl-dev           12.4.127             h6a678d5_0    defaults
cuda-profiler-api         12.4.127             h06a4308_1    defaults
cuda-runtime              12.4.1               hb982923_0    defaults
cuda-sanitizer-api        12.4.127             h99ab3db_1    defaults
cuda-toolkit              12.4.1               hb982923_0    defaults
cuda-tools                12.4.1               h06a4308_1    defaults
cuda-version              12.4                 hbda6634_3    defaults
cuda-visual-tools         12.4.1               h06a4308_1    defaults
dbus                      1.13.18              hb2f20db_0    defaults
expat                     2.6.2                h6a678d5_0    defaults
fontconfig                2.14.1               h4c34cd2_2    defaults
freetype                  2.12.1               h4a9f257_0    defaults
gcc_impl_linux-64         11.2.0               h1234567_1    defaults
gcc_linux-64              11.2.0               h5c386dc_0    defaults
gds-tools                 1.9.1.3              h99ab3db_1    defaults
glib                      2.78.4               h6a678d5_0    defaults
glib-tools                2.78.4               h6a678d5_0    defaults
gmp                       6.2.1                h295c915_3    defaults
gxx_impl_linux-64         11.2.0               h1234567_1    defaults
gxx_linux-64              11.2.0               hc2dff05_0    defaults
icu                       73.1                 h6a678d5_0    defaults
kernel-headers_linux-64   3.10.0              h57e8cba_10    defaults
krb5                      1.20.1               h143b758_1    defaults
ld_impl_linux-64          2.38                 h1181459_1    defaults
libcublas                 12.4.5.8             h99ab3db_1    defaults
libcublas-dev             12.4.5.8             h99ab3db_1    defaults
libcufft                  11.2.1.3             h99ab3db_1    defaults
libcufft-dev              11.2.1.3             h99ab3db_1    defaults
libcufile                 1.9.1.3              h99ab3db_1    defaults
libcufile-dev             1.9.1.3              h99ab3db_1    defaults
libcurand                 10.3.5.147           h99ab3db_1    defaults
libcurand-dev             10.3.5.147           h99ab3db_1    defaults
libcusolver               11.6.1.9             h99ab3db_1    defaults
libcusolver-dev           11.6.1.9             h99ab3db_1    defaults
libcusparse               12.3.1.170           h99ab3db_1    defaults
libcusparse-dev           12.3.1.170           h99ab3db_1    defaults
libedit                   3.1.20230828         h5eee18b_0    defaults
libffi                    3.4.4                h6a678d5_1    defaults
libgcc-devel_linux-64     11.2.0               h1234567_1    defaults
libgcc-ng                 11.2.0               h1234567_1    defaults
libglib                   2.78.4               hdc74915_0    defaults
libgomp                   11.2.0               h1234567_1    defaults
libiconv                  1.16                 h5eee18b_3    defaults
libnpp                    12.2.5.30            h99ab3db_1    defaults
libnpp-dev                12.2.5.30            h99ab3db_1    defaults
libnvfatbin               12.4.127             h7934f7d_2    defaults
libnvfatbin-dev           12.4.127             h7934f7d_2    defaults
libnvjitlink              12.4.127             h99ab3db_1    defaults
libnvjitlink-dev          12.4.127             h99ab3db_1    defaults
libnvjpeg                 12.3.1.117           h6a678d5_1    defaults
libnvjpeg-dev             12.3.1.117           h06a4308_1    defaults
libpng                    1.6.39               h5eee18b_0    defaults
libstdcxx-devel_linux-64  11.2.0               h1234567_1    defaults
libstdcxx-ng              11.2.0               h1234567_1    defaults
libuuid                   1.41.5               h5eee18b_0    defaults
libxcb                    1.15                 h7f8727e_0    defaults
libxkbcommon              1.0.1                h5eee18b_1    defaults
libxml2                   2.10.4               hfdd30dd_2    defaults
ncurses                   6.4                  h6a678d5_0    defaults
nsight-compute            2024.1.1.4           h968f9c8_2    defaults
nspr                      4.35                 h6a678d5_0    defaults
nss                       3.89.1               h6a678d5_0    defaults
ocl-icd                   2.3.2                h5eee18b_1    defaults
openssl                   3.0.14               h5eee18b_0    defaults
pcre2                     10.42                hebb0a14_1    defaults
pip                       24.0             py39h06a4308_0    defaults
python                    3.9.19               h955ad1f_1    defaults
readline                  8.2                  h5eee18b_0    defaults
setuptools                69.5.1           py39h06a4308_0    defaults
sqlite                    3.45.3               h5eee18b_0    defaults
sysroot_linux-64          2.17                h57e8cba_10    defaults
tk                        8.6.14               h39e8969_0    defaults
tzdata                    2024a                h04d1e81_0    defaults
wheel                     0.43.0           py39h06a4308_0    defaults
xz                        5.4.6                h5eee18b_1    defaults
zlib                      1.2.13               h5eee18b_1    defaults
```

### Additional information

Even when I specified different cuda version, it is installing only cuda 12.4.

**Comment by JCBreath** [comment]
It happens to me too. very annoying

**Comment by saundramonroe** [comment]
The warning nvcc warning : incompatible redefinition for option 'compiler-bindir', the last value of this option was used typically indicates that the compiler-bindir option has been defined multiple times with different values in your build process, and the last one defined is the one that will be used.

Can you check to see if your build scripts (like Makefile, CMakeLists.txt, or any custom build scripts) have multiple definitions of the **compiler-bindir** option and ensure that it is defined only once.

**Comment by ElectronicElephant** [comment]
This happens to me too. No matter what version specified, it always installs cuda 12.4. I have tested it on a new installed Ubuntu 20.04 and miniconda 24.5.0.

**Comment by JCBreath** [comment]
I finally solved this by using -c to specify the channel
`conda install -c nvidia/label/cuda-11.6.2 cuda-toolkit`

**Comment by zhxieml** [comment]
I encountered the same issue. It appears that using `conda install {channel}::{package}` assigns a lower priority to `channel` compared to `defaults`, causing version specifications to fail. In contrast, `conda install -c {channel} {package}` grants the highest priority and successfully applies the version specification. Have no idea why they changed the installation instructions to favor the former method...

**Comment by chenyaofo** [comment]
It may works: `conda install cuda -c nvidia/label/cuda-11.8.0`. Indeed, in the official guidance, it use the command in this way.

https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#conda-installation
